README.zh.md

```diff
@@ -29,6 +35,16 @@ Damn Vulnerable Web Application (DVWA) 的发布是为了希望它有用，但
 
 - - -
 
+## 国际化
+
+该文件有多种语言版本： 
+
+- 中文: [简体中文](README.zh.md)
```

Comment:
Maybe in the Chinese REAME, link the English version instead: 

- [English](README.md)